CHANGES

```diff
@@ -7,6 +7,8 @@ Dependencies
 Incompatible changes
 --------------------
 
+* The URL of download files are changed
```

Comment:
After this change, all download files are placed at `{outdir}/_downloads/{hash}/{filename}`.
The `hash` is generated from its pathname. So it would be changed the location of srcdir.

Is there better idea?